### ISSUE TITLE
Documentation issue "Getting Started" regarding "Docker Local Install"

### ISSUE THREAD

**Issue by xthk** [DefectDojo/django-DefectDojo#469]
### Issue Type 
- [X] Bug
- [ ] Enhancement

### Description

The docs at https://defectdojo.readthedocs.io/en/latest/getting-started.html#docker-local-install indicate that credentials are shown in the terminal. Unfortunately there are no credentials shown.

> Navigate to: http://localhost:8000 and login with the credentials shown in the terminal.

The expect script (https://github.com/DefectDojo/django-DefectDojo/blob/master/docker/setup-superuser.expect#L6-L7) contains the credentials (user: admin, password: admin).

Two options: Change [docs/getting-started.rst](https://github.com/DefectDojo/django-DefectDojo/blob/master/docs/getting-started.rst) to include the credentials or print the credentials to the terminal.

### Install

- [ ] setup.bash
- [X] docker

### Steps to Reproduce

```bash
docker pull appsecpipeline/django-defectdojo
docker run -it -p 8000:8000 appsecpipeline/django-defectdojo bash -c "export LOAD_SAMPLE_DATA=True && bash /django-DefectDojo/docker/docker-startup.bash"
```

### Console Logs (optional)

```
$ docker pull appsecpipeline/django-defectdojo
Using default tag: latest
latest: Pulling from appsecpipeline/django-defectdojo
Digest: sha256:4c1c12c52e604755d85b6c2da1f38d02558f4a9144a245bd6340ff78fa49ca74
Status: Image is up to date for appsecpipeline/django-defectdojo:latest
$ docker run -it -p 8000:8000 appsecpipeline/django-defectdojo bash -c "export LOAD_SAMPLE_DATA=True && bash /django-DefectDojo/docker/docker-startup.bash"
 * Starting MySQL database server mysqld                                 [ OK ]
```

**Comment by J12934** [comment]
Hi, can we reopen this?
This still seems to be happening and is a major hurdle in testing out defectdojo 😕

**Comment by tenderitaf** [comment]
any update on this ?

**Comment by AjCaroline** [comment]
wow this is frustrating.....Can someone help us?? How are we supposed to use this tool?? Where is the support for this? Anyone....????

**Comment by danielvonwi** [comment]
Local install doesn't seem to be a method of choice anymore (there is no local install section in the docs) and I assume it has been replaced with the Docker Compose method which is using a different repo defectdojo/defectdojo-django instead of appsecpipeline/django-defectdojo. 
Did you try the defectdojo/defectdojo-django which is also more recent than the appsecpipeline one

**Comment by asadasivan** [comment]
@danielvonwi I tried installing defect-dojo by cloning the repo ( DefectDojo /
django-DefectDojo) but the password doesn't seem to work. It says invalid password. Looks like MySQL is having some issues in changing the password for Admin. I also tried installing using the setup.bash script where I am able to login using the password displayed on the terminal but it is missing files or directories under static like bootstrap, Jquery, flot, etc.
